src/gui/pedalboard/BoardComponent.cpp

```diff
@@ -165,7 +168,7 @@ void BoardComponent::duplicateProcessor (const ProcessorEditor& editor)
     procChain.getProcStore().duplicateProcessor (*editor.getProcPtr());
 }
 
-void BoardComponent::showNewProcMenu (PopupMenu& menu, PopupMenu::Options& options)
+void BoardComponent::showNewProcMenu (PopupMenu& menu, PopupMenu::Options& options, bool generatedfromCableClick, Cable* clickedCable)
```

Comment:
I think `generatedFromCableClick` might be a redundant argument here, since you could just check`clickedCable == nullptr` and get the same thing?

Reply:
Yeah totally, thanks for catching that. I had the same realization yesterday.